packages/adminpanel-v2/src/components/sidebar/Sidebar.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+    import 'iconify-icon'
+
+    export let page_name : string 
+    export let page_route : string
+    export let icon : string
+</script>
+
+<a href={page_route} class="Sidebar-entry no-underline hover:underline">
+    <iconify-icon icon={icon} class="text-alv-dark-blue" width="30px"></iconify-icon>
```

Comment:
Suggestion 4 future: Not to use explicit pixel values, instead use rem/em/text-sm etc.